projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-update-post-endpoint.php

```diff
@@ -780,10 +780,10 @@ public function write_post( $path, $blog_id, $post_id ) {
 					}
 				}
 
-				$unslashed_meta_key           = wp_unslash( $meta->key ); // should match what the final key will be.
-				$meta->key                    = wp_slash( $meta->key );
-				$unslashed_existing_meta_key  = wp_unslash( $existing_meta_item->meta_key );
-				$existing_meta_item->meta_key = wp_slash( $existing_meta_item->meta_key );
+				$unslashed_meta_key           = wp_unslash( isset( $meta->key ) ? $meta->key : null ); // should match what the final key will be.
+				$meta->key                    = wp_slash( isset( $meta->key ) ? $meta->key : null );
+				$unslashed_existing_meta_key  = wp_unslash( isset( $existing_meta_item->meta_key ) ? $existing_meta_item->meta_key : null );
+				$existing_meta_item->meta_key = wp_slash( isset( $existing_meta_item->meta_key ) ? $existing_meta_item->meta_key : null );
```

Comment:
I think the isset makes more sense to be outside the slashing/unslashing, but very minor

Reply:
Thanks!